dapp/tests/utils.ts

```diff
@@ -99,6 +103,62 @@ export async function requestFaucetTokens(recipient: string) {
     }
 }
 
+interface TransactionToCreateAuctionParams {
+    sharedState: SharedState;
+    useNewWallet?: boolean;
```

Comment:
pass here the keypair directly:
```suggestion
    signer: Signer
```